.travis.yml

```diff
@@ -6,6 +6,7 @@ notifications:
   email: true
 node_js:
   - '6'
+  - '4'
```

Comment:
since you're purpose is to also support node v4, don't you think it's also a good thing to specify `"node": ">=4"` in `engines` field of `package.json`?